### ISSUE TITLE
fix shell completion

### ISSUE THREAD

**Issue by jkowalleck** [k4cg/nichtparasoup#226]
TL;DR: fix shell completion for version 3.
its currently disabled for a reason.

----

in upcoming verrsion 3 of _nichtparasoup_ 
the CLI will probably be driven via `click` instead of `argparse`

unfortunately `click` has a broken feature for shell autocompletion in version 7. 
* file/path completions not implemented 
  *  https://github.com/pallets/click/pull/1403
  * https://github.com/pallets/click/issues/780
* named subcommands broken 
  * https://github.com/pallets/click/pull/1529
it's fortunate that this will be fixed in version 8 - as far as the milestones give insignt.

an intermediate fix fia `click_completion` is incomplete, too: file completions are not working.
* file/path completions not implemented
* no option to manually implement them, since `click`'s `autocompletion` funciton is not taken into account
  * https://github.com/click-contrib/click-completion/pull/27
  * https://github.com/click-contrib/click-completion/pull/34

**Comment by jkowalleck** [comment]
there was an attempt to use alternative click completion, already.
see https://github.com/k4cg/nichtparasoup/blob/218cc230024aa34facd8fc2e237b22ae8b639245/src/nichtparasoup/cli/completion._py
